source/jormungandr/jormungandr/autocomplete/kraken.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+
+# Copyright (c) 2001-2016, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from jormungandr.autocomplete.abstract_autocomplete import AbstractAutocomplete
+from jormungandr.scenarios.utils import build_pagination, pb_type
+from jormungandr.interfaces.v1.fields import NonNullList, place, NonNullNested, PbField, error, feed_publisher,\
+    disruption_marshaller
+from flask.ext.restful import marshal_with, fields
+import navitiacommon.request_pb2 as request_pb2
+import navitiacommon.type_pb2 as type_pb2
+from jormungandr.utils import date_to_timestamp
+
+
+places = {
+    "places": NonNullList(NonNullNested(place)),
+    "error": PbField(error, attribute='error'),
+    "disruptions": fields.List(NonNullNested(disruption_marshaller), attribute="impacts"),
+    "feed_publishers": fields.List(NonNullNested(feed_publisher))
+}
+
+
+class Kraken(AbstractAutocomplete):
+
+    @marshal_with(places)
+    def get(self, request, instance):
```

Comment:
We have "Type [], admin_uri[]" argments, we can not specify the arguments. it can be a second PR.